cli/cmd/status.go

```diff
@@ -95,5 +107,6 @@ func statusCheckResultWasError(w io.Writer) error {
 
 func init() {
 	RootCmd.AddCommand(statusCmd)
+	statusCmd.PersistentFlags().BoolVar(&statusForceSystemInfo, "print-system-info", false, "Fores command to print system information, even if tests were successful.")
```

Comment:
Typo here: `Fores` => `Forces`. But I don't think this flag is checked anywhere. I'd probably just remove it.